docs/articles/expensify-classic/expensify-card/Statements.md

```diff
@@ -1,5 +1,75 @@
+
+We'll automatically withdraw this settlement amount from the Verified Business Bank Account linked to the primary domain admin. You can set up this bank account in the web app under Settings > Account > Payments > Bank Accounts.
+
+Once the payment is made, your Expensify Card balance will be $0, and the transactions are considered "settled."
+
+To change your settlement frequency or bank account, go to Settings > Domains > [Domain Name] > Company Cards. On the Company Cards page, click the Settings tab, choose a new settlement frequency or account from the dropdown menu, and click Save to confirm the change.
+
+![Change your card settlement account or settlement frequency via the dropdown menus in the middle of the screen.](https://help.expensify.com/assets/images/ExpensifyHelp_CardSettings.png){:width="100%"}
+
+# Expensify Card Statement and Settlements FAQs
+## **Can you pay your balance early if you've reached your Domain Limit?**
+  If you've chosen Monthly Settlement, you can manually initiate settlement using the "Settle Now" button. We'll settle the outstanding balance and then perform settlement again on your selected predetermined monthly settlement date.
+  If you opt for Daily Settlement, the Expensify Card statement will automatically settle daily through an automatic withdrawal from your business bank account. No additional action is needed on your part.
+  
+## **Will our domain limit change if our Verified Bank Account has a higher balance?**
+  Your domain limit may fluctuate based on your cash balance, spending patterns, and history with Expensify. Suppose you've recently transferred funds to the business bank account linked to Expensify card settlements. In that case, you should expect a change in your domain limit within 24 hours of the transfer (assuming your business bank account is connected through Plaid).
+  
+## **How is the “Amount Owed” figure on the card list calculated?**
+  The amount owed consists of all Expensify Card transactions, both pending and posted, since the last settlement date. The settlement amount withdrawn from your designated Verified Business Bank Account only includes posted transactions.
+  Your amount owed decreases when the settlement clears. Any pending transactions that don't post timely will automatically expire, reducing your amount owed.
+  
+## **How do I view all unsettled expenses?**
+  To view unsettled expenses since the last settlement, use the Reconciliation Dashboard's Expenses tab. Follow these steps:
+  1. Note the dates of expenses in your last settlement.
+  2. Switch to the Expenses tab on the Reconciliation Dashboard.
+  3. Set the start date just after the last settled expenses and the end date to today.
+  4. The Imported Total will show the outstanding amount, and you can click through to view individual expenses.
+
+
```

Comment:
Please delete these blank lines (leave just one blank line at the bottom)

Reply:
Deleted all but one blank line at the bottom!